.gitignore

```diff
@@ -38,6 +38,7 @@ test/truffle/cexts/**/mkmf.log
 
 # Maven
 lib/truffleruby.jar
+target
```

Comment:
This should be `/target` to match only at the root and not in any sub-directory.